x-pack/plugins/security_solution/scripts/endpoint/agent_emulator/services/endpoint_response_actions.ts

```diff
@@ -282,6 +287,7 @@ type ResponseOutput<TOutputContent extends object = object> = Pick<
 >;
 
 const getOutputDataIfNeeded = (action: ActionDetails): ResponseOutput => {
+  const endpointActionGenerator = new EndpointActionGenerator();
```

Comment:
yeah, same here.  Can you share why you needed to make changes to the tool? was there some issue you were encountering?